frappe/__init__.py

```diff
@@ -1857,7 +1857,7 @@ def get_list(doctype, *args, **kwargs):
 	        frappe.get_list("ToDo", fields="*", filters = [["modified", ">", "2014-01-01"]])
 
 	        # filter as a list of dicts
-	        frappe.get_list("ToDo", fields="*", filters = {"description": ("like", "test%")})
```

Comment:
You sure? I don't think this works :)

it should be just "filter as dictionary" and not list of dicts.

Reply:
so we should remove the `	        # filter as a list of dicts` part ?

Reply:
if it does not work

Reply:
yes, change it to "filter as dictionary"